migrations.lock

```diff
@@ -588,8 +588,7 @@ hqwebapp
  0009_truncate_authtoken_table
  0010_maintenancealert_scheduling
  0011_rename_maintenancealert_commcarehqalert
- 0012_commcarehqalert_created_by_domain
- 0013_commcarehqalert_created_by_user
+ 0012_auto_20231006_1052
```

Comment:
Didn't merge further with `0011_rename_maintenancealert_commcarehqalert` considering that migration was doing a rename which felt like it deserved to be kept as a single migration.

Reply:
renaming was avoided due to avoid unexpected issues during deploy. Only new fields were added and the "model" name was changed